From 8286399b9e1988b18810d762c4665658bb7428f9 Mon Sep 17 00:00:00 2001
From: Oswald Buddenhagen <oswald.buddenhagen@qt.io>
Date: Tue, 31 Jan 2017 11:19:16 +0100
Subject: [PATCH] fix installation of header-only frameworks

in a framework build, the headers are inside the "library", so it's
obviously not very wise to suppress its installation on the basis of
it not being there.

Task-number: QTBUG-57656
Change-Id: I026a3e486a2aad6ee0b8e0d264af4385af945e42
Reviewed-by: Jake Petroules <jake.petroules@qt.io>
Reviewed-by: Harald Fernengel <harryf@gmx.com>
---
 mkspecs/features/qt_module.prf | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/qtbase/mkspecs/features/qt_module.prf b/qtbase/mkspecs/features/qt_module.prf
index f6cbf99..ec31e45 100644
--- a/qtbase/mkspecs/features/qt_module.prf
+++ b/qtbase/mkspecs/features/qt_module.prf
@@ -68,9 +68,9 @@ load(qt_build_paths)
 
 header_module {
     TEMPLATE     = aux
-    CONFIG      += \
-        force_qt \  # Needed for the headers_clean tests.
-        qt_no_install_library
+    CONFIG      += force_qt  # Needed for the headers_clean tests.
+    !lib_bundle: \
+        CONFIG  += qt_no_install_library
 } else {
     TEMPLATE     = lib
 }
-- 
2.7.4

